package.json

```diff
@@ -66,7 +66,9 @@
   },
   "scripts": {
     "test": "mocha -R spec && npm run coverage",
-    "coverage": "rimraf cov-pt* && istanbul cover --report none --dir cov-pt0 node_modules/mocha/bin/_mocha && istanbul report --include cov-pt\\*/coverage.json",
+    "precoverage": "rimraf coverage && rimraf cov-pt*",
+    "coverage": "istanbul cover --report none --dir cov-pt0 node_modules/mocha/bin/_mocha -- -R dot",
```

Comment:
Actually I get why you put it there now. We already have a dedicated `test` script with proper output.